pkgs/applications/office/dspdfviewer/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitHub
+, boost
+, libsForQt5
+, qt5
+, cmake
+, pkg-config
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dspdfviewer";
+  version = "1.15.1";
+
+  src = fetchFromGitHub {
+    owner = "dannyedel";
+    repo = "dspdfviewer";
+    rev = "v${version}";
+    hash = "sha256-NDjbr7kWQP1OkikveLWlGCZzJUF0u1+dnzaeE32MK1g=";
+  };
+
+  cmakeFlags = [
+    "-DDCMAKE_BUILD_TYPE=Release"
+    "-DUsePrerenderedPDF=ON"
+    "-DCMAKE_CXX_FLAGS_INIT=\"-Wno-error=deprecated-declarations\" "
```

Comment:
Is upstream aware of these deprecation issue? If not, it'd be nice to notify them, and in any case put a link to an upstream issue.